core/rspamd/conf/classifier-bayes.conf

```diff
@@ -0,0 +1,6 @@
+autolearn {
+  spam_threshold = 6.0; # When to learn spam (score >= threshold)
```

Comment:
This is too strict. 12 is the default spam threshold score (80% * 15). It should not be 6 but 12.

Reply:
Strict? Or lenient?

Somewhat OT: as for the default 80% - from the experience I had with mailu so far I find it somewhat too high - I'm currently using 50% and it seems to hit the good spot (but probably YMMV)

Reply:
I've used the default values suggested by upstream (https://github.com/rspamd/rspamd/blob/master/conf/statistic.conf#L47)

You can't have it too high... otherwise it won't learn anything (as the email will either get rejected outright or put in the spam folder -where the sieve filter will kick in anyways-). If it doesn't learn spam there won't be room for it to learn ham; The whole idea is to make it re-enforce scores... the caveat is of course that unattended mailboxes may end up making it learn/re-enforce the wrong thing.

IMHO 6 is fine

Reply:
rephrasing the above, it's consistent with https://github.com/rspamd/rspamd/blob/e70bc00333e7639ac5f05bce2893221a589e451a/conf/actions.conf#L18 (that we don't override in Mailu).

If we rewrite the subject it means we consider it's spam. Unless the user re-classifies, it makes sense to learn from it and ensure that similar emails get a worse score in the future.

Reply:
Ok. Let's give it a try. We can always tweak this in the future.